code/game/machinery/doors/firedoor.dm

```diff
@@ -88,11 +94,236 @@
+
+		//TODOKYLER: the progress bar is inaccurate for teaming up, and only the first person will succeed, this needs to be centralized to work
+		///players can team up to open it faster. 20 seconds -> 13.333 -> 8.88 ... -> 2
+		if(do_after_dynamic(user, TRUE_OPENING_TIME, src, extra_checks = is_closed_callback, interaction_key = DOAFTER_SOURCE_BAREHAND_FIRELOCK_OPEN, max_interact_count = 1,  dynamic_timer_change = timer_callback))
+			user.visible_message("<span class='notice'>[living_user] opens [src] with [user.p_their()] [hand_string].</span>", \
+				self_message = "<span class='notice'>You pry open [src] with your [hand_string]!</span>")
+
+			barehanded_open_log(TRUE, user)
+
+			open()
+
+		else //user failed to open it
+			barehanded_open_log(FALSE, user)
+
+		people_trying_to_open -= user_stat_list
+		UnregisterSignal(user, COMSIG_PARENT_PREQDELETED)
+		user.move_resist = initial(user.move_resist)
+
+/obj/machinery/door/firedoor/attack_paw(mob/living/user, list/modifiers)
+	. = ..()
+	if(!user.combat_mode)
+		attack_hand(user, modifiers)
+
+/**
+ * deals with logging either when failing or succeeding to open a firelock without a crowbar.
+ * everyone trying to open it are logged individually, regardless of whether theres more than one trying to do so
+ * * success - whether or not the user opened it successfully
+ * * user - the person who either succeeded or failed in opening it, returns if not specified
+ */
+/obj/machinery/door/firedoor/proc/barehanded_open_log(success, mob/living/user)
+	if(QDELETED(user) || !isliving(user))
+		return
+
+	var/list/user_stat_list
+
+	var/user_opening_duration
+
+	var/end_temperature_of_user
+
+	var/user_damage_taken
+
+	var/user_has_internals = "false"
+
+	for(var/list/user_list as anything in people_trying_to_open)
+		if(user_list["user"] == user)
+			user_stat_list = user_list
+			break
+
+	if(!user_stat_list)
+		return
+
+	if(iscarbon(user))
+		var/mob/living/carbon/carbon_user = user
+		if(carbon_user.internal)
+			user_has_internals = "true"
+	user_opening_duration = (REALTIMEOFDAY - user_stat_list["starting time"]) / 10
+
+	user_damage_taken = user.health - user_stat_list["starting health"]
+
+	var/datum/gas_mixture/user_environment = user.return_air()
+	end_temperature_of_user = user_environment.temperature
+
+	if(success || (!density && user.stat == CONSCIOUS)) //if we're open and the user is conscious, then no matter what we're called with its a success
+		SSblackbox.record_feedback("associative", "barehand firelock opens", 1, \
+		list( \
+		"user" = key_name(user), \
+		"starting temperature" = user_stat_list["starting temperature"], \
+		"ending temperature" = end_temperature_of_user, \
+		"user damage taken" = user_damage_taken, \
+		"size of group trying to open" = length(people_trying_to_open), \
+		"user has internals on" = user_has_internals \
+		))
```

Comment:
Indent properly.

The `list(` and the final `)` should be up one, and the contents should be up another. Also, all these backslashes are unnecessary.